src/ucs/datastruct/bitmap.h

```diff
@@ -94,7 +257,7 @@ _UCS_BITMAP_DECLARE_TYPE(256)
  * @return Bit value (0 or 1)
  */
 #define UCS_BITMAP_GET(_bitmap, _index) \
-    (!!(_UCS_BITMAP_WORD(_bitmap, _index) & \
+    (!!(_UCS_BITMAP_WORD_BY_BIT(_bitmap, _index) & \
```

Comment:
maybe rename _index to _bit_index in all places to show where is a word index and where is a bit index